data_processing/common/Node.py

```diff
@@ -9,18 +9,22 @@ class Node(object):
 	:param: name: required node name. e.g. scan-123
 	:param: properties: dict of key, value pairs for the node.
 	"""
-	def __init__(self, node_type, node_name, properties={}):
+	def __init__(self, node_type, node_name, properties=None):
```

Comment:
The dictionary in this case gets tied to the class namespace, and persisted across multiple class instances